pkg/sloop/webfiles/resource.html

```diff
@@ -12,14 +12,19 @@
     <link rel="stylesheet" type="text/css" href="/webfiles/sloop.css">
     <link rel="stylesheet" type="text/css" href="/webfiles/resource.css">
     <link href="https://unpkg.com/nprogress@0.2.0/nprogress.css" rel="stylesheet" />
+    <link rel="stylesheet" type="text/css" href="http://cemerick.github.io/jsdifflib/diffview.css"/>
+
     <title>Resource {{.Kind}}/{{.Namespace}}/{{.Name}}</title>
 
     <script src="https://unpkg.com/axios/dist/axios.min.js" type="text/javascript"></script>
     <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js" type="text/javascript"></script>
     <script src="https://unpkg.com/nprogress@0.2.0/nprogress.js" type="text/javascript"></script>
+    <script type="text/javascript" src="http://cemerick.github.io/jsdifflib/diffview.js"></script>
+    <script type="text/javascript" src="http://cemerick.github.io/jsdifflib/difflib.js"></script>
```

Comment:
nit: http->https
please also move type after src just for consistency with includes above